third_party/rust-hypervisor-firmware-virtio/src/device.rs

```diff
@@ -136,6 +136,11 @@ where
         self.transport.read_device_config(offset)
     }
 
+    /// Gets the device status.
```

Comment:
Do we have a link to the status codes it may return?

Reply:
Added link to the spec.